apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -116,9 +138,11 @@ async fn create_client(
 struct RouteService {
     host: String,
     route_table: &'static RwLock<HashMap<PathBuf, RouteMeta>>,
+    consumption_apis: &'static RwLock<HashSet<String>>,
```

Comment:
I am not a big fan of that mutable state punched through. Not sure that there is a better solution for now though